src/pages/Event/Event.layout.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <detail-layout :image_url="event.image_url">
+    <template #custom-name>
```

Comment:
What is this #custom-name

Reply:
The name is confused since #custom-name sounds like an example lol. But it's a attribute that allow VueJS to pass multiple children on a component, if exists two slots, for example:

```jsx
<header>
    <slot name="header"></slot>
</header>
<main>
    <slot name="body"></slot>
</main>
```

we can access to him using `<template #{name}></template>`

for example 
```
<component>
    <template #header>This is a header</template>
    <template #body>This is a body</template>
</component>
```

This is the documentation for it:

https://vuejs.org/guide/components/slots.html#named-slots

We can too use v-slot:{name} instead,

```
<component>
    <template v-slot:header>This is a header</template>
    <template v-slot:body>This is a body</template>
</component>
```

Using `#` is just the shorthand syntax